app/mailers/patient_mailer.rb

```diff
@@ -72,9 +73,17 @@ def assessment_sms_weblink(patient)
         add_success_history(dependent, patient)
       else
         add_fail_history_sms(dependent)
+        failed_dependents = true
         patient.update(last_assessment_reminder_sent: nil) # Reset send attempt timestamp on failure
       end
     end
+    return unless patient.dependents.count >= 2 && !patient.active_dependents.include?(patient)
```

Comment:
My only question is why we notify the HoH of the failed message sent to their dependent _only_ in the case where the HoH is not an active_dependent?
ie: 
-
Active HoH -Success
HoH Active Dependent -Fail
-
Would result in a success history on the HoH and a fail on the Dependent

Whereas:
-
InActive HoH - N/A
HoH Active Dependent -Fail
-
Would result in a fail on the HoH and a fail on the Dependent.

Seems to me you would want to have a history item on the HoH notifying them of failed messages regardless of if the HoH themselves are active or not.